sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/CosmosConstants.java

```diff
@@ -5,7 +5,7 @@
 
 import com.azure.core.util.CoreUtils;
 
-public class KafkaCosmosConstants {
+public class CosmosConstants {
```

Comment:
Isn't there already a `CosmosConstants.scala` class? May be better to keep it with Kafka prefix or suffix?

Reply:
updated